app/connectors/httpParsers/MultiplePenaltiesHttpParser.scala

```diff
@@ -45,6 +46,9 @@ object MultiplePenaltiesHttpParser {
         case NO_CONTENT =>
           logger.debug(s"$startOfLogging $NO_CONTENT returned from penalties backend")
           Left(NoContent)
+        case CONFLICT =>
```

Comment:
Could you add a test in MultiplePenaltiesHttpParserSpec, please? Similar to how you have done it in AppealSubmissionParserSpec